Python 3.11 - Swagger._handle_swagger_method_call() missing 1 required positional argument: 'route'

In python3.11, the following tests fail: 
```
FAILED tests/test_class_based_view.py::test_class_based_view[pyloop] - AssertionError: assert 500 == 200
FAILED tests/test_class_based_view.py::test_decorated_class_based_view[pyloop] - AssertionError: assert 500 == 200
FAILED tests/test_class_based_view.py::test_class_based_spec_file[pyloop] - AssertionError: assert 500 == 200
FAILED tests/test_docs_decorator.py::test_decorated_handlers[pyloop] - AssertionError: assert 500 == 200
FAILED tests/test_methods.py::test_all_methods[pyloop] - AssertionError: assert 500 == 200
```

For all of them, the exception in the web server is:

```
ERROR    aiohttp.server:web_protocol.py:403 Error handling request
Traceback (most recent call last):
  File ".../lib64/python3.11/site-packages/aiohttp/web_protocol.py", line 433, in _handle_request
    resp = await request_handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib64/python3.11/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File ".../lib64/python3.11/site-packages/aiohttp/web_urldispatcher.py", line 956, in _iter
    resp = await method()
                 ^^^^^^^^
TypeError: Swagger._handle_swagger_method_call() missing 1 required positional argument: 'route'
```

This is happening due to what seems to be a change in behavior of `partialmethod` used [here](https://github.com/hh-h/aiohttp-swagger3/blob/v0.7.2/aiohttp_swagger3/swagger_docs.py#L158).    I've written up a minimal test case that demonstrates this change [here](https://discuss.python.org/t/context-manager-for-asyncio-tasks/14609).


I've got 2 ideas how to fix this in. PR's to follow shortly.

--- a/aiohttp_swagger3/swagger.py
+++ b/aiohttp_swagger3/swagger.py
@@ -132,14 +132,6 @@ class Swagger(web.UrlDispatcher):
 
         self._app[ui_index_html] = ui_template.substitute({"settings": json.dumps(ui_settings.to_settings())})
 
-    async def _handle_swagger_call(self, route: "SwaggerRoute", request: web.Request) -> web.StreamResponse:
-        kwargs = await route.parse(request)
-        return await route.handler(**kwargs)
-
-    async def _handle_swagger_method_call(self, view: web.View, route: "SwaggerRoute") -> web.StreamResponse:
-        kwargs = await route.parse(view.request)
-        return await route.handler(view, **kwargs)
-
     def add_head(self, path: str, handler: WebHandler, **kwargs: Any) -> web.AbstractRoute:
         return self.add_route(hdrs.METH_HEAD, path, handler, **kwargs)
 
@@ -235,3 +227,13 @@ class Swagger(web.UrlDispatcher):
                 raise Exception(f"register handler for {media_type} first")
             return self.handlers[typ]["*"]
         return self.handlers[typ][subtype]
+
+
+async def _handle_swagger_call(route: "SwaggerRoute", request: web.Request) -> web.StreamResponse:
+    kwargs = await route.parse(request)
+    return await route.handler(**kwargs)
+
+
+async def _handle_swagger_method_call(view: web.View, route: "SwaggerRoute") -> web.StreamResponse:
+    kwargs = await route.parse(view.request)
+    return await route.handler(view, **kwargs)

--- a/aiohttp_swagger3/swagger_docs.py
+++ b/aiohttp_swagger3/swagger_docs.py
@@ -10,7 +10,7 @@ from aiohttp import hdrs, web
 from aiohttp.abc import AbstractView
 
 from .routes import _SWAGGER_SPECIFICATION
-from .swagger import ExpectHandler, Swagger
+from .swagger import ExpectHandler, Swagger, _handle_swagger_call, _handle_swagger_method_call
 from .swagger_info import SwaggerInfo
 from .swagger_route import SwaggerRoute, _SwaggerHandler
 from .ui_settings import RapiDocUiSettings, ReDocUiSettings, SwaggerUiSettings
@@ -155,8 +155,8 @@ class SwaggerDocs(Swagger):
             return handler
         route = SwaggerRoute(method, path, handler, swagger=self)
         if is_method:
-            return functools.partialmethod(self._handle_swagger_method_call, route)  # type: ignore
-        return functools.partial(self._handle_swagger_call, route)
+            return functools.partialmethod(_handle_swagger_method_call, route)  # type: ignore
+        return functools.partial(_handle_swagger_call, route)
 
     def add_route(
         self,

--- a/aiohttp_swagger3/swagger_file.py
+++ b/aiohttp_swagger3/swagger_file.py
@@ -6,7 +6,7 @@ from aiohttp import hdrs, web
 from aiohttp.abc import AbstractView
 
 from .routes import _SWAGGER_SPECIFICATION
-from .swagger import ExpectHandler, Swagger
+from .swagger import ExpectHandler, Swagger, _handle_swagger_call, _handle_swagger_method_call
 from .swagger_route import SwaggerRoute, _SwaggerHandler
 from .ui_settings import RapiDocUiSettings, ReDocUiSettings, SwaggerUiSettings
 
@@ -80,12 +80,12 @@ class SwaggerFile(Swagger):
                     setattr(
                         handler,
                         meth,
-                        functools.partialmethod(self._handle_swagger_method_call, route),
+                        functools.partialmethod(_handle_swagger_method_call, route),
                     )
             else:
                 method_lower = method.lower()
                 if method_lower in self.spec["paths"][path]:
                     route = SwaggerRoute(method_lower, path, handler, swagger=self)
-                    handler = functools.partial(self._handle_swagger_call, route)
+                    handler = functools.partial(_handle_swagger_call, route)
 
         return self._app.router.add_route(method, path, handler, name=name, expect_handler=expect_handler)

